### PR TITLE
Update Next.js from 9.5.3 to 9.5.5

### PR DESCRIPTION
**Pull request by wanjuntham**
Closes: #1274

### What are the changes and their implications?

Updated nextjs to 9.5.4 for @blitzjs/server package.

### Checklist

- [ ] Tests added for changes
- [ ] PR submitted to [blitzjs.com](https://github.com/blitz-js/blitzjs.com) for any user facing changes

### CODE REVIEW

**Comment by wanjuntham** [comment]
Could you please advise me what changes I need to further make to pass CI?

**Comment by flybayer** [comment]
Thanks @wanjuntham! `9.5.5` was just released, so can you go ahead and change this to that?

And the CI... another ugly windows test thing rears it's head. This was broke, then worked fine for awhile. It's not related to your changes. I'll try to get it fixed.

**Comment by wanjuntham** [comment]
> 9.5.5 was just released, so can you go ahead and change this to that?

Done https://github.com/blitz-js/blitz/pull/1289/commits/0e7423b3e4e96419cbc0f052bb1cd270b81f4b14

> It's not related to your changes.

Ahh okay good to hear.

1 more question here, please confirm if I need to commit `package-lock.json` file, as I dont see this file before I do `npm install`.

File: packages/server/package.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
     "lodash-es": "^4.17.15",
     "merge-stream": "2.0.0",
     "nanoid": "3.1.10",
-    "next": "9.5.3",
+    "next": "^9.5.5",
```
Comment (flybayer) [inline packages/server/package.json:51]: This needs to be an exact version number.

```suggestion
    "next": "9.5.5",
```

**Review by flybayer — CHANGES_REQUESTED** [review]
Cool thanks!

Yeah, package-lock.json should not be added. You need to run `yarn` instead of `npm install`.

File: packages/server/package.json (hunk 1 above)
Comment (wanjuntham) [reply packages/server/package.json]: Okay just did

**Comment by wanjuntham** [comment]
Done removing `package-lock.json`.

**Comment by flybayer** [comment]
@wanjuntham thanks, but now it's missing changes to `yarn.lock` which needs to be generated by running `yarn`.

**Comment by wanjuntham** [comment]
Done commiting latest yarn.lock file

**Comment by flybayer** [comment]
Sweet, thanks!!

@all-contributors add @wanjuntham for code

**Pull request merged**